ajax/libs/react-mdl/1.10.3/material.css

```diff
@@ -0,0 +1,11466 @@
+/**
+ * material-design-lite - Material Design Components in CSS, JS and HTML
+ * @version v1.2.1
+ * @license Apache-2.0
+ * @copyright 2015 Google, Inc.
+ * @link https://github.com/google/material-design-lite
```

Comment:
OK then maybe we don't have to add these files under extra?
https://cdnjs.com/libraries/material-design-lite/1.2.1
And maybe you don't have to add the v1.0.0 & v1.0.1 in gitignore.

What do you think?

Reply:
`extra` is required for this `react-mdl`. See https://github.com/tleunen/react-mdl#requirements
> Instead of using the official material.css and material.js files from the MDL project, you should use the files we provide in /extra.